domains/erickacevedor.json

```
@@ -0,0 +1,8 @@
+{
+  "owner": {
+    "username": "erickacevedor",
+    "email": "erickacevedor@gmail.com"
+  },
+  "URL": "https://tini.bio/erick",
+  "proxied": true
```

needs a records, proxied must be outside of "records" and you cant proxied the URL records
```suggestion
  "records": {
  "URL": "https://tini.bio/erick"
  }
```